funnel/static/css/app.css

```diff
@@ -845,7 +850,7 @@
 .schedule .schedule__row .schedule__row__column {
   padding: 5px 10px;
   position: relative;
-  background: url('../img/schedule-box-background.png?1544501585');
+  background: url('../img/schedule-box-background.png?1544526769');
```

Comment:
What is this ID?

Reply:
compass auto generates the url for caching purpose.
http://compass-style.org/reference/compass/helpers/urls/#image-url

Reply:
The id number changes when Compass detects that the referred file has changed. However, since this change detection is based on the filesystem (inode number, I think), if two developers work on the same stylesheet file, Compass will assume all the image files have changed as well. We will only solve this problem if we move build to an automated subsystem, or if Compass switches to an advanced change detection system, such as depending on Git to identify changed files.